src/firefly/java/edu/caltech/ipac/util/FitsHDUUtil.java

```diff
@@ -115,13 +115,16 @@ public static DataGroup fitsHeaderToDataGroup(String fitsFile) {
         DataGroup dg = new DataGroup("fits", cols);
         String invalidMsg = "invalid fits file";
 
+        FitsFactory.useThreadLocalSettings(true);
+        boolean prevLongStringEnabled = FitsFactory.isLongStringsEnabled();
```

Comment:
The use of `useTreadLocalSettings` is not needed here.  The added code can be removed.